website/docs/docs/cloud/migration.md

```diff
@@ -53,7 +53,7 @@ Complete all of these items to ensure your <Constant name="cloud" /> resources a
 
 Use one of these two URL login options:
 
-- `us1.dbt.com.` If you were previously logging in at `cloud.getdbt.com`, you should instead plan to log in at us1.dbt.com. The original URL will still work, but you’ll have to click through to be redirected upon login.
+- `us1.dbt.com.` If you were previously logging in with a username and password at `cloud.getdbt.com`, you should instead plan to log in at us1.dbt.com. The original URL will still work, but you’ll have to click through to be redirected upon login. If you have single sign-on configured, you will use the unique URL listed in the SSO account settings. 
```

Comment:
> you should instead plan to log in at us1.dbt.com

Do we still want to do this? Or should we be redirecting all users to login using their new post-migration access URL?

Reply:
@matthewshaver Right now, all of the accounts we are migrating live in US MT and will be moved into a US1 MC cell -- though this will change in the future.

Technically, users can login at `us1.dbt.com` after migration; however, this shows the regional account switcher, which is not very useful if they don't have accounts across multiple cells in the region. Also, they will have to auth a second time at the cellular level when choosing the account they want to access.